app/overrides/spree/admin/products/_form/add_group_buy_field.html.haml.deface

```diff
@@ -0,0 +1,3 @@
+/ replace "tr[data-hook='product_group_buy_unit_size_field']"
+
+= render 'spree/admin/products/group_buy_form', f: f
```

Comment:
I don't get why this file is needed. AFAIK we're just replacing hardcoded copies with I18n keys.

Reply:
Hi @sauloperez I was getting a duplicated group buy form on the page.. That's why I created this file in the first place. Now I've just deleted the file to check if I was still getting the duplication.. but the duplication has gone. Anyway.. it's all good now. Tks. Leandro. 👍 😄